projects/fal/src/fal/app.py

```diff
@@ -300,8 +354,26 @@ async def set_global_object_preference(request, call_next):
                     "Failed set a global lifecycle preference %s",
                     self.__class__.__name__,
                 )
+
             return await call_next(request)
 
+        @app.middleware("http")
+        async def set_request_id(request, call_next):
+            if self.isolate_channel is None:
+                self.isolate_channel = await open_isolate_channel(
+                    f"localhost:{get_grpc_port()}"
```

Comment:
sorry to be annoying but: we can just do `os.getenv` if you prefer now, this is only run once now